src/Sulu/Bundle/MediaBundle/Resources/js/containers/MediaCollection/MediaSection.js

```diff
@@ -20,11 +23,23 @@ export default class MediaSection extends React.PureComponent<Props> {
             adapters,
             listStore,
             mediaListRef,
+            onUploadOverlayOpen,
+            uploadable,
         } = this.props;
 
         return (
             <List
                 adapters={adapters}
+                customButtons={[
+                    uploadable && (
+                        <ButtonGroup key="upload-media">
+                            <Button icon="su-upload" onClick={onUploadOverlayOpen}>
+                                Upload File
```

Comment:
We should use translation here. `t('sulu_media.upload_file')` seems like a new translation which you wound need to add in the `admin.de.json` and `admin.en.json` of MediaBundle

Reply:
`t('sulu_media.upload')` should work too. Or adding `t('sulu_media.upload_files')` I guess, because multiple files are possible too.